client/src/components/Posts/Hero.tsx

```diff
@@ -43,12 +42,19 @@ const Hero = () => {
       {
         //@ts-ignore
         posts.map((item, index) => (
-          <div key={index} className="flex justify-center items-center">
-            <Card
+          <div key={index} className="flex gap-4 justify-center items-center m-3">
+            {/* <Card
```

Comment:
We need to have a <Card /> component and try to include the media player in the Card component and the source will be passed as the prop.

Reply:
done